src/mesh.rs

```diff
@@ -371,6 +381,25 @@ impl<'a> Primitive<'a> {
         self.json
     }
 
+    /// Returns the bounds (min/max) of the POSITION attribute if there is one, otherwise `None`.
+    /// May panic for invalid glTF files. Use json::validation::Validate::validate_minimally
+    /// to handle this gracefully.
+    pub fn bounds(&self) -> Option<Bounds> {
```

Comment:
I'd prefer to return a tuple `(min, max)` instead of an explicit `Bounds` type. It's up for debate though.

```rust
let (min, max) = primitive.bounds();
```